mllib/src/test/scala/org/apache/spark/ml/feature/RFormulaSuite.scala

```diff
@@ -143,6 +143,44 @@ class RFormulaSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(attrs === expectedAttrs)
   }
 
+  test("vector attribute generation") {
+    val formula = new RFormula().setFormula("id ~ vec")
+    val original = sqlContext.createDataFrame(
+      Seq((1, Vectors.dense(0.0, 1.0)), (2, Vectors.dense(1.0, 2.0)))
```

Comment:
I think it makes sense when using RFormula in a ML pipeline (not necessarily in R).